spec.bs

```diff
@@ -5354,6 +5353,21 @@ result of [=evaluating a bidding script=], or an [=additional bid=] provided by
   : <dfn>component seller</dfn>
   :: Null or an [=origin=]. Seller in component auction which the [=generated bid/interest group=]
     is participating. Only set for component auctions, null otherwise.
+</dl>
+
+A <dfn>bid debug reporting info</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="bid debug reporting info">
+  : <dfn>id</dfn>
+  :: An {{unsigned long}}-or-null. Matches a [=generated bid/id=] of the [=generated bid=] corresponding to
+     this reporting information, or null if no bid was produced, in which case the debug loss URLs are
+     still relevant.
+  : <dfn>component seller</dfn>
+  ::  Null or an [=origin=]. Seller in component auction which was running to produce
+    this debug information. Only set for component auctions, null otherwise.

```

Comment:
nit: "debug information" -> "reporting information", to be consistent. Can also link to [=bid debug reporting info=], but no strong preference on that.